docs/source/TaskEngineWorkflows.md

```diff
@@ -551,8 +590,11 @@ A jobs `run_at` attribute can be set in multiple ways and defaults to the time i
     "run_at": "1970-01-01T00:00:00"
   }
 ```
+If the jobs run_at time is in the future, then this will be logged against the job id and is viewable on the jobs log page.
```

Comment:
job's*

If the job's run_at time is in the future, a log will be added to indicate such. The log will be viewable from the job's log page.